controllers/component_image_controller.go

```diff
@@ -295,6 +257,41 @@ func (r *ComponentImageReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 	return ctrl.Result{}, nil
 }
 
+func getImageReferenceFromBuildPipeline(buildPipelineRun tektonapi.PipelineRun) (string, error) {
+	pipelineResults := buildPipelineRun.Status.PipelineResults
+	if pipelineResults == nil {
+		return "", fmt.Errorf("PipelineRun.Status.PipelineResults is empty")
+	}
+
+	imageUrl := ""
+	for _, pipelineRunResult := range pipelineResults {
+		if pipelineRunResult.Name == "IMAGE_URL" {
```

Comment:
who is creating theses pipelineruns @mmorhun , build-service or application-service?  

are constants for these names not defined in one of the repos?  or maybe the new application-api repo?

Reply:
Initial build pipeline is created by build-service. Next are created by Tekton or PaC.
The constants defined [here](https://github.com/redhat-appstudio/build-definitions/blob/main/pipelines/base/template-build/template-build.yaml#L99-L102).